interface/patient_file/report/custom_report.php

```diff
@@ -837,7 +844,15 @@ function postToGet($arin)
 
     if ($PDF_OUTPUT == 1) {
         try {
-            $pdf->Output($fn, $GLOBALS['pdf_output']); // D = Download, I = Inline
+            if ($PDF_FAX === 1) {
+                $fax_pdf = $pdf->Output($fn, 'S');
+                $tmp_file = $GLOBALS['OE_SITE_DIR'] . '/documents/temp/' . $fn;
```

Comment:
TODO
This is for me to go back to to ensure the temp file is removed after complet review

Reply:
deletes in sendFax methods